pkg/meta/repodb/boltdb-wrapper/boltdb_wrapper.go

```diff
@@ -747,8 +733,33 @@ func (bdw *DBWrapper) AddManifestSignature(repo string, signedManifestDigest god
 		buck := tx.Bucket([]byte(bolt.RepoMetadataBucket))
 
 		repoMetaBlob := buck.Get([]byte(repo))
-		if repoMetaBlob == nil {
-			return zerr.ErrManifestMetaNotFound
+
+		if len(repoMetaBlob) == 0 {
+			var err error
+			// create a new object
+			repoMeta := repodb.RepoMetadata{
```

Comment:
So it doesn't matter if the image or the signature is added first? Does this change take care of my concern in the other comment?

Reply:
Yes, this change takes care of the case when the signature is added before the signe manifest